sysutils/dnscontrol/Portfile

```diff
@@ -29,7 +29,11 @@ maintainers         {@ajhall} \
 # Allow deps to fetched at build time
 build.env-delete    GO111MODULE=off GOPROXY=off
 
-build.args-append   -ldflags \"-s -w\"
+build.args-append   -ldflags \"-s -w \
+                                -X main.Version=${version} \
+                                -X main.BuildTime=[clock seconds] \
```

Comment:
MacPorts prefers that build time fields usually be set to some constant value like or "unset", "", "unknown", or "n/a" for the sake of reproducible builds.  We still have this behavior in quite a few ports still (and also within the makefiles and build processes of many projects), but the preference is to reduce instances of setting build time dynamically across ports.

Reply:
Ah, makes sense. It works fine if that value isn't specified, so I just removed it.